tests/ps_test.go

```diff
@@ -88,7 +88,7 @@ var _ = Describe("Processes", func() {
 			Entry("scales to 3", 3, 200),
 			Entry("scales to 0", 0, 502),
 			Entry("scales to 5", 5, 200),
-			Entry("scales to 0", 0, 502),
+			PEntry("scales to 0", 0, 502),
 			Entry("scales to 1", 1, 200),
```

Comment:
We'll need to `P` this line as well until fix supplied for related issue https://github.com/deis/workflow-e2e/issues/69